e2e-tests/cypress/support/common/assertIs400ErrorPage.js

```diff
@@ -0,0 +1,17 @@
+const defaultOptions = {
+  expectedTitle: 'GOV.UK - The best place to find government services and information',
+  expectedMessage: '',
+};
+
+module.exports = ({
+  expectedTitle = defaultOptions.expectedTitle,
+  expectedMessage = defaultOptions.expectedMessage,
+} = defaultOptions) => {
+  cy.title().should('eq', expectedTitle);
+
```

Comment:
Should indeed, but nuts as this may sound, this is non trivial in Cypress (afaik).

Cypress wants to check the status code using `cy.visit` - however, in doing so, it would throw / blow up if we hit a 400 page.

In this instance we do actually want to visit the 400 status page, which means we have to pass `{ failOnStatusCode: false }` to visit, at which point we dont get access to the status code any longer. 

I may be entirely wrong, or missing a trick, but from the reading up / googling I did, this wasn't possible.